ruby_programming/in_progress/oop_basics/oop_introduction_lesson.md

```diff
@@ -1,5 +1,7 @@
 ### Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+ So far you wrote programs executed line by line, in methods, as you learned them. It's called procedural programming. Now you think it's easier to understand how to make a program like that, not thinking about how your methods will be organized further, but later you will build real programs and applications. If your code will be organized that way, a single mistake, a typo, can make your build fall apart. Object-oriented approach propose you to organize your code into structures named for chosen object called CLASSES. 
```

Comment:
I think we need to simplify this to better explain some OOP concepts. A person with no programming experience won't know what classes or objects are.